.travis.yml

```diff
@@ -81,4 +81,5 @@ script:
   - tests/api_test
   - '[ "$TRAVIS_OS_NAME" == "osx" ] || tests/p2p_tests/sync/test.sh'
   - tests/p2p_tests/sync/test.sh -p 2 -d 10
+  - tests/run_tests.sh
```

Comment:
Change to: `'[ "$TRAVIS_OS_NAME" == "osx" ] || tests/run_tests.sh`
The mac build doesn't have enough time to run these tests.